src/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -72,23 +71,22 @@ public static void showDownloadMapsWindow() {
   }
 
 
-  private static void showDownloadMapsWindow(final Component parent, Optional<String> mapName) {
+  private static void showDownloadMapsWindow(final JFrame parent, Optional<String> mapName) {
     final DownloadRunnable download = new DownloadRunnable(ClientContext.mapListingSource().getMapListDownloadSite());
     final String popupWindowTitle = "Downloading list of availabe maps....";
     BackgroundTaskRunner.runInBackground(null, popupWindowTitle, download);
     final List<DownloadFileDescription> games = download.getDownloads();
     checkNotNull(games);
 
-    final Frame parentFrame = JOptionPane.getFrameForComponent(parent);
-    final DownloadMapsWindow dia = new DownloadMapsWindow(mapName, games);
+    final DownloadMapsWindow dia = new DownloadMapsWindow(mapName, games, parent);
     dia.setSize(800, WINDOW_HEIGHT);
-    dia.setLocationRelativeTo(parentFrame);
+    dia.setLocationRelativeTo(parent);
     dia.setMinimumSize(new Dimension(200, 200));
     dia.setVisible(true);
   }
-
-  private DownloadMapsWindow(final Optional<String> mapName, final List<DownloadFileDescription> games) {
-    super("Download Maps");
+  
+  private DownloadMapsWindow(final Optional<String> mapName, final List<DownloadFileDescription> games, JFrame parent) {
+    super(parent, "Download Maps", true);
```

Comment:
if this true is changed to false or removed the download maps windows will keep independent....